src/styles/card.css

```
@@ -0,0 +1,92 @@
+.card {
+  color: var(--dark-color);
+}
+
+.card__container {
+  max-width: 200px;
+  padding: 32px 16px 0;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
```

move styles to card and remove margin bottom from btn and add to padding
```suggestion
  width: 200px;
  padding: 32px 16px 0; // why 0?

  border: 1px solid #f3f3f3;
  border-radius: 5px;
```